cmd/utils.go

```diff
@@ -326,6 +326,12 @@ func UTCNowNTP() (time.Time, error) {
 	return ntp.Time(ntpServer)
 }
 
+func timeToMsDosTime(t time.Time) (fDate uint16, fTime uint16) {
+	fDate = uint16(t.Day() + int(t.Month())<<5 + (t.Year()-1980)<<9)
```

Comment:
What is it with 1980 ?

Reply:
Can we not do epoch for t.Unix()

Reply:
Year in MS-DOS is calculated using `1980` as the offset year. It is `1970` for Unix.
So, t.Unix() is not going to work for MS-DOS.

Reply:
Yes that I already saw but we shouldn't do this on our end, we should wait for @klauspost for his inputs.

Reply:
OK. We'll wait for @klauspost 's response then.